Gopkg.toml

```diff
@@ -10,38 +10,44 @@ required = [
   "k8s.io/gengo/args",
 ]
 
+[[override]]
+  name = "gopkg.in/fsnotify.v1"
+  source = "https://github.com/fsnotify/fsnotify.git"
+
 [[override]]
   name = "k8s.io/code-generator"
-  # revision for tag "kubernetes-1.12.3"
-  revision = "3dcf91f64f638563e5106f21f50c31fa361c918d"
+  # revision for tag "kubernetes-1.16.0"
+  revision = "cd179ad6a2693011d6f2fa5cd64c6680ee99379f"
```

Comment:
@squeed why did we do it this way rather than just doing `version = "kubernetes-1.16.0"` ?

Reply:
Cargo cullted from a long time ago. As I recall, client-go didn't used to tag k8s versions.

We can almost certainly switch to more meaningful tags now.

Reply:
i support the idea of switching to tags, way more human readable. Can amend later once I sort out CI issues